packages/components/src/VirtualizedList/RendererSelector.component.js

```diff
@@ -56,14 +56,14 @@ function RendererSelector(props) {
 				id={id}
 				isActive={isActive}
 				isSelected={isSelected}
+				noRowsRenderer={() => <props.noRowsRenderer />}
```

Comment:
If I refer to the Proptypes, this could be an element instead of a function. So could it be written like `noRowsRenderer={<props.noRowsRenderer />}`

Reply:
I tried to pass only the component `noRowsRenderer={props.noRowsRenderer}` but it failed saying that a class cannot be called like a function.
I haven't looked into react-virtualized to spot the line of code, but my assumption is that they just call the noRowsRenderer as a function, but our default NoRows is wrapped in a translate HOC that returns a class.

Reply:
noRowsRender need a function which 'll be called by react-virtualized